slack/command.py

```diff
@@ -98,11 +99,7 @@ def set(text, context):
         'url',
         'baseurl',
         'url_pattern',
-        'urlpattern',
```

Comment:
Why not go with a consistent scheme? like `baseurl` + `urlseparator`

Reply:
for that matter `baseurl` should probably be `basedomain`...

Reply:
would changing `baseurl` have backwards compatibility issues?

Reply:
> apart from the below, can you think of any other Slack command changes? I think there were a few that were weird - maybe `config` could also be something else?

One idea is a new Slack command `open`. Example: `/cimon open {ENVIRONMENT}`, and get webbrowser to open that environment..

Reply:
If the `config` command was to change, i would suggest make a generic `get` command. Example `/cimon get {url_pattern}` or `/cimon get {config}`

Reply:
I think if we changed `baseurl` we could change the allowed input but retain the keyword in the database settings - i.e. only change it on the interface side of things. 

Not sure how we'd get Slack top open a browser but we could at least print the URL to the conversation window. The plan originally was to post to the conversation automatically but I ran out of time (and I think that's harder in Slack than responding to a command).

Generic `get` could be good - might be nice to spit out all the settings with `get` though, as opposed to specifying each one?

Reply:
> I think if we changed `baseurl` we could change the allowed input but retain the keyword in the database settings - i.e. only change it on the interface side of things.
> 
> Not sure how we'd get Slack top open a browser but we could at least print the URL to the conversation window. The plan originally was to post to the conversation automatically but I ran out of time (and I think that's harder in Slack than responding to a command).
> 
> Generic `get` could be good - might be nice to spit out all the settings with `get` though, as opposed to specifying each one?

I believe my newest commit should avoid changing db keywords - let me know if not. I felt that snake case (`base_domain`) made more sense than no spaces in variable names, seems more in line with the codebase as a whole, and reduced the number of changes required like this https://github.com/signal-noise/deploybot/blob/7ea523605642bb8938b8d299d0a73ce7a389347e/slack/command.py#L199-L204

I've updated `config` command to `get`.

Reply:
I initially used snake case for the same reason, and it's the python convention. 
However I'm now leaning towards no breaks as it's simpler to type, and these commands / var names are effectively the app interface, which I think trumps codebase convention. 
Not a deal-breaker either way.